src/desktop/apps/artist2/components/__tests__/Meta.jest.js

```diff
@@ -0,0 +1,51 @@
+import { toJSONLD } from 'desktop/apps/artist2/components/Meta.js'
```

Comment:
Best to avoid adding extensions to `.js` / `.ts` files.

Reply:
This errors and shows not found without extension

Reply:
ooo really? That's strange... let me try it on my side / I wonder if it has something to do with jest setup

Reply:
@eessex - just ran a test by creating a new JS file that was imported from a Jest test and all ok. Mind pasting in the issue you're seeing? 

<img width="356" alt="screen shot 2018-08-28 at 12 51 36 pm" src="https://user-images.githubusercontent.com/236943/44747137-418d7600-aac1-11e8-9aaa-3975394092ba.png">

![screen shot 2018-08-28 at 12 51 38 pm](https://user-images.githubusercontent.com/236943/44747143-46eac080-aac1-11e8-9603-e85de85c2c27.png)